modules/core/react/lib/type.ts

```diff
@@ -2,7 +2,12 @@ import {default as colors, typeColors, statusColors} from '@workday/canvas-color
 import {borderRadius} from './radius';
 import {CSSProperties} from './types';
 
-export const fontFamily = '"Roboto", "Helvetica Neue", "Helvetica", Arial, sans-serif';
+const inheritFont =
+  // @ts-ignore
+  window.workday && window.workday.canvas && window.workday.canvas.inheritFontFamily;
```

Comment:
Seems like there should be a better way...

First, it seems wrong to do a `// @ts-ignore`. Using `lodash.get` would be a bit better here.

Second, we are referencing paths that exist within one product. It seems like we need a different approach to allow overrides with an official API from our side. Maybe through the theme?

Reply:
> First, it seems wrong to do a `// @ts-ignore`

[We're already doing this for `useTheme`](https://github.com/Workday/canvas-kit/blob/master/modules/_labs/core/react/lib/theming/useTheme.ts#L34). The idea was that we don't want to add to the window global scope for people who aren't using this functionality. Adding these fields (i.e. `declare global { interface Window { ... }}`) would add them for all consumers. `lodash.get` would probably work if you think that's preferable.

> Second, we are referencing paths that exist within one product. It seems like we need a different approach to allow overrides with an official API from our side. Maybe through the theme?

The idea is that anyone could specify this window variable. My approach in https://github.com/Workday/canvas-kit/pull/552 was more of an official API using theming, but the performance was worse, it allowed consumers to specify _any_ font (which we want to avoid), and every bespoke team in product would need to add it to their theme object. The method in this PR is a one and done thing controlled by the UI Platform team for any apps within the Workday SPA.